### DISCUSSION TITLE
PaddleOCR/main/ppocr/infer_deploy/paddle_server

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleOCR#14208] [Q&A]
# PaddleOCR/main/ppocr/infer_deploy/paddle_server

Awesome multilingual OCR toolkits based on PaddlePaddle (practical ultra lightweight OCR system, support 80+ languages recognition, provide data annotation and synthesis tools, support training and deployment among server, mobile, embedded and IoT devices)

https://paddlepaddle.github.io/PaddleOCR/main/ppocr/infer_deploy/paddle_server.html

**Comment by asenasen123** [comment]
请问下pdserving方式部署时，如果要用到表格识别模型，如何修改config.yml？
